docs/coordinates/galactocentric.rst

```diff
@@ -0,0 +1,113 @@
+.. _coordinates-galactocentric:
+
+==========================================
+Transforming to Galactocentric coordinates
+==========================================
+
+This document describes the mathematics behind the transformation from
+:class:`~astropy.coordinates.ICRS` to `~astropy.coordinates.Galactocentric`
+coordinates. For examples of how to use this transformation in code, see
+the *Examples* section of the
+`~astropy.coordinates.Galactocentric` class documentation.
+
+We assume that we start with a 3D position in the ICRS reference frame:
+a Right Ascension, Declination, and heliocentric distance,
+:math:`(\alpha, \delta, d)`. We can trivially convert this to a
+Cartesian position using the standard transformation from Cartesian to
+spherical coordinates:
+
+.. math::
+
+   \begin{aligned}
+       x_{\rm icrs} &= d\cos{\alpha}\cos{\delta}\\
+       y_{\rm icrs} &= d\sin{\alpha}\cos{\delta}\\
+       z_{\rm icrs} &= d\sin{\delta}\\
+       \boldsymbol{r}_{\rm icrs} &= \begin{pmatrix}
+         x_{\rm icrs}\\
+         y_{\rm icrs}\\
+         z_{\rm icrs}
+       \end{pmatrix}\end{aligned}
+
+The first transformations will rotate the :math:`x_{\rm icrs}` axis so
+that the new :math:`x'` axis points towards the Galactic Center (GC),
+specified by the ICRS position
+:math:`(\alpha_{\rm GC}, \delta_{\rm GC})`:
+
+.. math::
+
+   \begin{aligned}
+       \boldsymbol{R}_1 &= \begin{bmatrix}
+         \cos\delta_{\rm GC}& 0 & -\sin\delta_{\rm GC}\\
+         0 & 1 & 0 \\
+         \sin\delta_{\rm GC}& 0 & \cos\delta_{\rm GC}\end{bmatrix}\\
+       \boldsymbol{R}_2 &=
+       \begin{bmatrix}
+         \cos\alpha_{\rm GC}& \sin\alpha_{\rm GC}& 0\\
+         -\sin\alpha_{\rm GC}& \cos\alpha_{\rm GC}& 0\\
+         0 & 0 & 1
+       \end{bmatrix}.\end{aligned}
+
+The transformation thus far has aligned the :math:`x'` axis with the
+vector pointing from the Sun to the GC, but the :math:`y'` and
+:math:`z'` axes point in an arbitrary direction. We adopt the
+orientation of the Galactic plane as the normal to the north pole of
+Galactic coordinates defined by the IAU
+(`Blaauw et. al. 1960 <http://adsabs.harvard.edu/abs/1960MNRAS.121..164B>`_).
+This extra “roll” angle, :math:`\eta`, was measured by transforming a grid
+of points along :math:`l=0` to this interim frame and minimizing the square
+of their :math:`y'` positions. We find:
+
+.. math::
+
+   \begin{aligned}
+       \eta &= 148.5986320^\circ\\
+       \boldsymbol{R}_3 &=
+       \begin{bmatrix}
+         1 & 0 & 0\\
+         0 & \cos\eta & \sin\eta\\
+         0 & -\sin\eta & \cos\eta
+       \end{bmatrix}\end{aligned}
+
+The full rotation matrix thus far is:
```

Comment:
@cdeil I disagree -- sorry -- I'd prefer to leave it all in there.

Will oblige if others have opinions? @eteq @astrofrog